AnnoyancesFilter/sections/popups.txt

```diff
@@ -109,6 +109,7 @@ jeotermalhaberler.com#$##subscribeModal { display: none!important; }
 jeotermalhaberler.com#$#.modal-backdrop { display: none!important; }
 jeotermalhaberler.com#$#body { overflow: visible!important; }
 glavkniga.ru##.popup_banner_area
+vanityfair.com##.persistent-bottom
```

Comment:
I don't think that we should block it. @AdguardTeam/filters-maintainers any thoughts on that?

Reply:
There is also a "Subscribe" button on the top, so I think that we can block it.
It's annoying in my opinion.

Reply:
I agree that is really annoying. Ok, let's block it, if others don't mind.